### DISCUSSION TITLE
stat-computing-r-python/part-gen-prog/04-control-struct

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#6] [General]
# stat-computing-r-python/part-gen-prog/04-control-struct

https://srvanderplas.github.io/stat-computing-r-python/part-gen-prog/04-control-struct.html

**Comment by extrastats** [comment]
An example of a loop or if-else statement that also utilizes strings (instead of just numeric data) would be very helpful. Especially in the palindrome homework!

**Comment by TylerBrown123** [comment]
The YouTube video about the “make a peanut butter sandwich instructions challenge” appears to not be working because it says "This video is private."
